CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @dsvoronin @eugene-krivobokov @sboishtyan @int02h
```

Comment:
@int02h , is it ok for you?

We can mark only performance modules

Reply:
Did you mean @warabei14 ?

Reply:
Yes